build_projects/Microsoft.DotNet.Cli.Build.Framework/Enumerations/BuildArchitecture.cs

```diff
@@ -3,6 +3,8 @@ namespace Microsoft.DotNet.Cli.Build.Framework
     public enum BuildArchitecture
     {
         x86 = 1,
-        x64 = 2
+        x64 = 2,
+        arm = 3,
+        arm64 = 4
```

Comment:
Please see https://github.com/dotnet/core-setup/blob/master/build_projects/dotnet-host-build/build.ps1#L39 - targetplatform gets sets to TargetArch which will be Arm64.